Functions/Quantification/xASL_quant_FSL.m

```diff
@@ -430,6 +483,60 @@
 					fprintf(FIDoptionFile, '--te%d=%.3f\n', iTE, TEs(iTE));
 				end
 			end
+            
```

Comment:
WIll review this function later.

Reply:
In general, I think that we should do this a bit more elegant. Especially since this will be more tricky for merging 3 files. I propose that in the FSL-options function we do the following steps:
1. We read fsloptions.txt at the start of the function and we read it for all previous sessions.
2. We parts all fsloptions and derive all important parameters from it - PLDs etc and put them in a separate character arrays.
3. We write a function that takes a character array and increases all numbers in a form PLD_XX etc by a given number so transforming PLD_1, PLD_2 to e.g. PLD_3, PLD_4. This function will only take the field format and the increment. We also write a function that obtains the last position in that field -> for PLD, it would return 4.
4. When merging, we always concatenate all the previous values, we do the necessary incrementing, and then we add the new values with a further increment.

All this will make it much easier and more transparent and will allow merging also 3 sessions - which is probably not possible now. Or at least I have only checked the code quickly, but haven't noticed any field increments.

Reply:
So right now, I have improved xASL_quant_FSL, it's far from perfect, but it is able to work with a merged list of PLDs. Still, this merging happens inside xASL_quant_FSL, but after merging #997, we will have the PWI4D.jsons and the merged JSON and we'll read all the info from there. So it will all become really general.

Reply:
I would still separate the merging function now, that will save us time. The PWI4D.jsons will be done in #1543 indeed but the separate function should be done here.

Reply:
What's our final decision? Just to make sure? To keep as is now and more details will be added to the issue and code as comments? So we can resolve this now?